### PR TITLE
feat(core): allow skipped stages to be omitted from the execution graph

### PR DESCRIPTION
**Pull request by anotherchrisberry**
If you end up skipping a bunch of stages, you can have a pretty sparse graph, e.g.
screen shot 2018-03-01 at 10 19 06 am

With this change, we can omit those stages if they are not relevant:
screen shot 2018-03-01 at 10 19 24 am

### CODE REVIEW

File: app/scripts/modules/core/src/domain/IExecution.ts [hunk 1]
```diff
@@ -21,7 +21,6 @@ export interface IExecution extends IOrchestratedItem {
   pipelineConfigId?: string;
   searchField?: string;
   stageSummaries?: IExecutionStageSummary[]; // added by transformer
-  stageWidth?: string; // added by transformer
```
Comment (anotherchrisberry) [inline app/scripts/modules/core/src/domain/IExecution.ts]: We don't use this field any more.

**Comment by christopherthielen** [comment]
should we put this "hide skipped stages" toggle on the pipeline level?

**Comment by anotherchrisberry** [comment]
@christopherthielen good question. For my use case, it would make sense to have it at the pipeline level. But there's a discoverability issue - do we put it in the Configuration section of the pipeline? Or should we make it a pipeline-level option, but show it whenever a stage is made conditional?

**Comment by christopherthielen** [comment]
I'd probably vote for "the Configuration section of the pipeline".

**Comment by anotherchrisberry** [comment]
Updated to "the Configuration section of the pipeline".

**Comment by anotherchrisberry** [comment]
Depends on https://github.com/spinnaker/orca/pull/2039 to get the pipeline-level flag set

**Comment by anotherchrisberry** [comment]
How about we don't do this.

**Comment by christopherthielen** [comment]
ok, but why not?

**Pull request closed without merging**